test/public/logs/overview.test.js

```diff
@@ -120,7 +124,7 @@ module.exports = () => {
             // eslint-disable-next-line no-undef
             model.logs.overviewModel.reset();
         });
-        await waitForTimeout(100);
+        await page.waitForFunction(() => document.querySelectorAll('table tbody tr').length === 10, { timeout: 1500 });
```

Comment:
Same as before, you can directly wait for table to have a non-loading row

Reply:
Done